Clients/webfrontauth-ngx/projects/webfrontauth-ngx/src/lib/AuthInterceptor.ts

```diff
@@ -14,6 +14,7 @@ export class AuthInterceptor implements HttpInterceptor {
     public intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
         return next.handle(
             this.authService.token !== ''
+            && this.authService.shouldSetToken(request.url!)
                 ? request.clone({headers: request.headers.set('Authorization', 'Bearer ' + this.authService.token)})
```

Comment:
Le check de this.authService.token !== '' on devrait pas le faire une bonne fois pour toute dans shouldSetToken ?